packages/lit-virtualizer/src/LitVirtualizer.ts

```diff
@@ -12,9 +12,9 @@ import { Virtualizer, VirtualizerHostElement, virtualizerRef, RangeChangedEvent
 import { LayoutSpecifier, Layout, LayoutConstructor } from './layouts/shared/Layout.js';
 
 
-type RenderItemFunction = ((item: any, index: number) => TemplateResult)
-const defaultKeyFunction = (item: any) => item;
-const defaultRenderItem: RenderItemFunction = (item: any, idx: number) => html`${idx}: ${JSON.stringify(item, null, 2)}`;
+type RenderItemFunction = (<T>(item: T, index: number) => TemplateResult)
```

Comment:
oh that's interesting, I didn't know you could declare the type parameter on the right hand side of the assignment like that!

Reply:
I really love that the solution you came up with was refining the typings of the generated directive function.  It never occurred to me that was possible, but it's so clearly the right answer answer to the problem at hand.  This PR is surely one to grow on.

![one to grow on](https://c.tenor.com/6im6Psayfc8AAAAC/one-to-grow-on-betty-white.gif)